_drafts/conda.md

```diff
@@ -0,0 +1,698 @@
+managers and I strongly advise against using this approach.
+
+#### Virtualenv
+
+Virtualenv works well for managing Python environments and packages, but it
+doesn't manage system binaries. It also requires that each Python version be
+installed prior to being used in a virtual environment, such as with `pyenv` or
+the system package manager.
+
+You can use Virtualenv alongside Conda as well if that is necessary, but it is
+not required. Conda is a full replacement for Virtualenv.
+
+#### Virtual Machine
+
+Using a virtual machine is a great way to isolate the environment even more so
+that every developer uses a similar environment and is provided with a GUI
+and/or IDE. It's also especially useful if everyone has a Linux machine and the
+build requires Windows, or vice-versa. However, I still find issues with this
+approach.
+
+- Sharing file systems between the host and image has performance issues.
+- Using the UI within a virtual machine is generally slower.
+- One still would need some sort of environment management with the VM, as
+  packages will inevitably change every couple of months.
+
+If a VM is chosen, I suggest using Conda **within it** to help manage the
+packages that are used within the VM. Otherwise, the team would have to package
+and distribute a new VM image each time a dependency changes, or hope that
+everyone knows when and how to update the system (and does it).
+
+## Example Conda Walk-through
+
+In this part of the post, we will set up Conda on our local machine and set up
+an environment that is able to build, flash, and debug an nRF52 device.
+
+If you'd like to follow along, you may download the Interrupt repo and navigate
+to the example directory.
+
+```
+$ git clone git@github.com:memfault/interrupt.git
+$ cd interrupt/example/using-conda/
+```
+
+> Before we begin, I want to bring comfort to anyone skeptical about installing
+> yet another tool. Conda is the opposite of invasive, and everything is
+> installed into a single directory, `$HOME/miniconda3` by default. If you want
+> to uninstall Conda, deleting that directory will be all that is necessary.
+
+### Installing Conda
+
+To install the `conda` client on a machine, the recommended way is to download
+the interactive
+[Miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it
+through `bash`. The following steps apply to both Linux and macOS. If you are
+using Windows, there is a Miniconda installer that can be downloaded from the
+same page.
+
+```
+$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+$ bash Miniconda3-latest-MacOSX-x86_64.sh
+```
+
+The final step in the installer is to make sure the
+`$HOME/miniconda3/etc/profile.d/conda.sh` script is loaded in the shell. When
+the following prompt is given:
+
+```
+Do you wish the installer to initialize Miniconda3
+by running conda init? [yes|no]
+```
+
+I suggest saying `no` and modifying the appropriate shell startup file
```

Comment:
Why?

Reply:
Adding an actual explanation here. Thanks for pointing that out.